packages/lodestar/src/util/graffiti.ts

```diff
@@ -0,0 +1,12 @@
+import {GRAFFITI_SIZE} from "../constants";
+
+/**
+ * Parses a graffiti ASCII string and returns a 32 bytes buffer right padded with zeros
+ * @param graffiti
+ */
+export function toGraffitiBuffer(graffiti: string): Buffer {
+  return Buffer.concat([
+    Buffer.from(graffiti, "ascii"),
```

Comment:
I think we should allow for utf-8. seems that block explorers are parsing the graffiti as utf-8 and hex.
As long as we slice to 32 bytes, we're fine.